src/Publisher.ts

```diff
@@ -311,7 +311,13 @@ export default class Publisher {
             try{
                 const block = queryBlock[0];
                 const query = queryBlock[1];
-                const markdown = await dvApi.tryQueryMarkdown(query, path);
+                const {isInsideCallout, finalQuery} = this.sanitizeQuery(query)
+                let markdown = await dvApi.tryQueryMarkdown(finalQuery, path);
+
+                if (isInsideCallout) {
+                    const tmp = markdown.split("\n");
+                    markdown = " " + tmp.join("\n> ")
```

Comment:
As the exact same logic is used in two places, would it perhaps be better to refactor this into its own function. Something like "surroundWithCalloutBlock".